examples/provision.rs

```diff
@@ -18,7 +18,6 @@ async fn main() -> anyhow::Result<()> {
     // Create a new account. This will generate a fresh ECDSA key for you.
     // Alternatively, restore an account from serialized credentials by
     // using `Account::from_credentials()`.
-
```

Comment:
Ahh, I kinda like the whitespace for this.

Reply:
NP. I'll revert this.